components/01-atoms/typography/headings/_headings.scss

```diff
@@ -7,7 +7,7 @@ h4,
 h5,
 h6 {
   color: var(--color-heading);
-  margin-block: 0;
+  margin-block: 0 0.6em;
```

Comment:
Similar comment as below. We don't have an exact spacing match but `--size-spacing-3` or `--size-spacing-4` may be a better option?

Reply:
In this case, no. We're using `em` because we want the value to adjust relative to the font size. So the larger the font gets, the larger the margin.